lib/rubocop/config_loader.rb

```diff
@@ -139,7 +139,10 @@ def add_excludes_from_files(config, config_file)
         exclusion_file = find_last_file_upwards(DOTFILE, config_file, ConfigFinder.project_root)
 
         return unless exclusion_file
-        return if PathUtil.relative_path(exclusion_file) == PathUtil.relative_path(config_file)
+
+        exclusion_relative_path = PathUtil.relative_path(exclusion_file)
+        return if exclusion_relative_path == PathUtil.relative_path(config_file)
+        return if exclusion_relative_path == PathUtil.relative_path(File.join(Dir.home, DOTFILE))
```

Comment:
Can you refactor to the below?

```suggestion
        return if exclusion_relative_path == PathUtil.relative_path(config_file) ||
                  exclusion_relative_path == PathUtil.relative_path(File.join(Dir.home, DOTFILE))
```